README.md

```diff
@@ -112,10 +106,111 @@ SuperGood::SolidusTaxjar.tap do |config|
 end
 ```
 
+### Settings
+
+Developers can configure the following settings in an initializer:
+
+- `cache_duration`: The length of time that cacheable responses from
+  the TaxJar API are cached. Cacheable responses include the list of active
+  nexus regions, as well as tax and tax rate calculations.
+
+  Default value: `3.hours`
+
+- `cache_key`:  function that returns a distinct cache key for
+  a cacheable response from the TaxJar API. A safe default function is provided.
+
+  Default value: (See the source code.)
+
+- `custom_order_params`: A function that returns any parameterized custom
+	order information you need to send to TaxJar in every API request that
+	includes a customer's order. By default no custom order parameters are
+	sent.
+
+	Note that all the essential `Spree::Order`, `Spree::LineItem`, and
+	`Spree::Address` attributes are already sent by default.
+
+	Default value: `{}`
+
+- `discount_calculator`: A discount calculator class. A safe default
+  calculator is provided, but may not be suitable for
+  stores. Check with your accountant how discounts need
+  to be calculated.
+
+  Default value: `::SuperGood::SolidusTaxjar::DiscountCalculator`
+
+- `exception_handler`: A function that returns your own exception
+  handler. See [Exception handling](#exception-handling).
+
+  Default value: (See the source code.)
+
+- `job_queue`: A string or symbole that matches the name of a
+  job queue in your application's job queuer. (The extension uses ActiveJob to
+  delegate new jobs.)
+
+	Default value: `:default`
+
+- `line_item_tax_label_maker`: A string to label line item taxes in TaxJar.
+
+  Default value: "Sales Tax"
+
+- `logging_enabled`: Boolean. Whether logging is enabled.
+
+  Default value: `true`
+
+- `shipping_calculator`: A function that calculates the total cost of the
+  shipments for an order. A default default is provided
+  inline, but your store may require additional logic
+  for accurate calculations.
+
+  Default value: (See the source code.)
+
+- `shipping_tax_label_maker`: A string to label shipment taxes in TaxJar.
+
+  Default value: "Sales Tax"
+
+- `taxable_address_check`: A function that returns a boolean after checking
+  whether an address should be considered taxable.  By default this function
```

Comment:
After [this PR](https://github.com/SuperGoodSoft/solidus_taxjar/pull/205), the default will be true unless you are in the united states and the address is **not** in a nexus region

Reply:
Okay. I will leave as is, and if this gets merged first I can add a commit to that PR.

Reply:
@forkata and I just double-checked. We've refactored the linked PR and we no longer change the `taxable_address_check`'s default return value.